pkgs/applications/graphics/inkscape/extensions/silhouette/default.nix

```diff
@@ -0,0 +1,78 @@
+{ fetchFromGitHub
+, lib
+, pkgs
+, python3
+, umockdev
+, writeScript
+}:
+
+let
+  launch-sendto_silhouette = writeScript "sendto_silhouette.sh" ''
+    cd $(dirname $0)
+    ./sendto_silhouette.py "$@"
+  '';
+  launch-silhouette_multi = writeScript "silhouette_multi.sh" ''
+    cd $(dirname $0)
+    ./silhouette_multi.py "$@"
+  '';
```

Comment:
Could you add a comment why this is necessary? If using the original `.py` files with shell does not work for some reason.

Reply:
Oh thanks for pointing this out. This is definitely no obvious. Commented added!